server_admin/topics/roles.adoc

```diff
@@ -1,9 +1,4 @@
 == Roles
+A role is a category of user. An application can assign access and permissions to a role and then assign multiple users to that rule, so that the users all share the same access and permissions. For example, the Admin Console has roles that give permission to users to access parts of the Admin Console UI.
 
-Roles identify a type or category of user.  `Admin`, `user`, `manager`, and `employee` are all typical roles that may exist
-in an organization.
-Applications often assign access and permissions to specific roles rather than individual users as dealing
-with users can be too fine grained and hard to manage.  For example, the Admin Console has specific roles which give permission to users
-to access parts of the Admin Console UI and perform certain actions.  There is a global namespace for roles and each client also has its own
-dedicated namespace where roles can be defined.
-
+There is a global namespace for roles and each client also has its own dedicated namespace where roles can be defined. Typical roles in an organization include `Admin`, `user`, `manager`, and `employee`. 
```

Comment:
I think it works best if you follow the ""A role typically applies to one type of user." sentence with the "Typical roles..." sentence.